pkg/constants/constants.go

```diff
@@ -25,8 +25,8 @@ const (
 	QueueAnnotation = "kueue.x-k8s.io/queue-name"
 
 	KueueName         = "kueue"
-	ManagerName       = KueueName + "-manager"
 	JobControllerName = KueueName + "-job-controller"
+	AdmissionName     = KueueName + "-admission"
```

Comment:
https://kubernetes.io/docs/reference/using-api/server-side-apply/#field-management

Every field of the object has a manager, even if you didn't "Apply".
If you don't specify the manager through CreateOptions or UpdateOptions, the manager is extracted from the UserAgent (hence #336).

I wanted to distinguish things that are applied by the admission logic, rather than kueue as a whole.